src/dep/triemap/triemap.h

```diff
@@ -45,7 +45,8 @@ typedef struct {
 
 typedef struct {
   TrieMapNode *root;
-  size_t cardinality;
+  size_t cardinality : 32;  // number of terms
```

Comment:
Why limit it to 32 bit? it's not like we have so many tries that it will consume too much memory no?